### PR TITLE
*: notify snapshot stats immediately.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing @zhangjinpeng1987

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -323,13 +323,15 @@ fn build_raftkv(matches: &Matches,
     db_path.push("db");
     let engine = Arc::new(DB::open(&opts, db_path.to_str().unwrap()).unwrap());
 
+    let mut event_loop = store::create_event_loop(&cfg.store_cfg).unwrap();
+
```
Comment (ngaut) [inline src/bin/tikv-server.rs]: Remove blank line.

File: tests/raftstore/test_stats.rs [hunk 2]
```diff
@@ -138,9 +137,10 @@ fn must_detect_snap(pd_client: &Arc<TestPdClient>) {
     for _ in 0..200 {
         sleep_ms(10);
 
-        let stats = pd_client.get_store_stats(1).unwrap();
-        if stats.get_snap_sending_count() > 0 || stats.get_snap_receiving_count() > 0 {
-            return;
+        if let Some(stats) = pd_client.get_store_stats(1) {
+            if stats.get_snap_sending_count() > 0 || stats.get_snap_receiving_count() > 0 {
```
Comment (ngaut) [inline tests/raftstore/test_stats.rs]: s/get_snap_sending_count/get_sending_snap_count/

Comment (ngaut) [inline tests/raftstore/test_stats.rs]: s/get_snap_receiving_count/get_receiving_snap_count/

**Comment by BusyJay** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL

**Comment by ngaut** [comment]
LGTM

**Pull request merged**